package NYT.NSequoiaServer.NProto;

import "yt_proto/yt/core/misc/proto/guid.proto";

////////////////////////////////////////////////////////////////////////////////

message TReqFlushQueue
{
    required int64 start_sequence_number = 1;
    required int64 end_sequence_number = 2;
}

////////////////////////////////////////////////////////////////////////////////
